recipes/assertpy/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
```

Comment:
Use jinja templating variables as shown in the example:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L6